skynet/modules/ttt/rag/vector_store.py

```diff
@@ -40,9 +40,7 @@ def bypass_ingestion(config: RagConfig, new_config: RagConfig):
 
 
 class SkynetVectorStore(ABC):
```

Comment:
Why is that no longer needed?

Reply:
cause the model we're gonna switch to is backed by the standard sentence transformers

Reply:
Then can we update the env var's default value, since it won't work otherwise?